index.bs

```diff
@@ -3705,6 +3688,27 @@ To <dfn>trigger event-level attribution</dfn> given an [=attribution trigger=] |
              ("<code>[=trigger debug data type/trigger-event-low-priority=]</code>", |report|)).
 
     </dl>
+1. Let |rateLimitRecord| be a new [=attribution rate-limit record=] with the items:
+    : [=attribution rate-limit record/scope=]
+    :: "<code>[=rate-limit scope/event-attribution=]</code>"
+    : [=attribution rate-limit record/source site=]
+    :: |sourceToAttribute|'s [=attribution source/source site=]
+    : [=attribution rate-limit record/attribution destination=]
+    :: |trigger|'s [=attribution trigger/attribution destination=]
+    : [=attribution rate-limit record/reporting origin=]
+    :: |sourceToAttribute|'s [=attribution source/reporting origin=]
+    : [=attribution rate-limit record/time=]
+    :: |sourceToAttribute|'s [=attribution source/source time=]
+    : [=attribution rate-limit record/expiry time=]
+    :: null
+    : [=attribution rate-limit record/event-level report ID=]
+    :: |report|'s [=event-level report/report ID=]
+1. If the result of running [=check if attribution should be blocked by rate limits=]
+    with |trigger|, |sourceToAttribute|, and |rateLimitRecord| is not null, return it.
+
+    Note: When [=max attributions per rate-limit window=] has already been
```

Comment:
Do we still need this note which seems to be self-explained by the algorithm?

Reply:
Removed.